AnkiDroid/src/main/java/com/ichi2/anki/CardTemplatePreviewer.java

```diff
@@ -373,9 +388,18 @@ private Card setCurrentCardFromNoteEditorBundle(Collection col) {
         if (model == null) {
             return null;
         }
+        if(mAllFieldsNull)
+            getLabels(fieldValues);
         Note n = getCol().newNote(model);
         for (int i = 0; i < fieldValues.size() && i < n.getFields().length; i++) {
-            n.setField(i, fieldValues.get(i));
+            if(mAllFieldsNull)
+                if(mCardType != null && (mCardType.equals(getString(R.string.cloze_model_name)) && i == 0) ||
+                        (mCardType.equals(getString(R.string.basic_typing_model_name)) && i ==1))
+                    n.setField(i, fieldValues.get(i));
+                else
+                    n.setField(i, "(" + fieldValues.get(i) + ")");
+            else
+                n.setField(i, fieldValues.get(i));
```

Comment:
```suggestion
            if (mAllFieldsNull) {
                if ((mCardType != null) && (mCardType.equals(getString(R.string.cloze_model_name)) && (i == 0)) ||
                        (mCardType.equals(getString(R.string.basic_typing_model_name)) && (i ==1))) {
                    n.setField(i, fieldValues.get(i));
                } else {
                    n.setField(i, "(" + fieldValues.get(i) + ")");
                }
            } else {
                n.setField(i, fieldValues.get(i));
            }
```

reformatting comments per our coding standards for java - 
- https://github.com/ankidroid/Anki-Android/wiki/Code-style#1tbs-one-true-brace-style-must-be-used-that-means
- https://github.com/ankidroid/Anki-Android/wiki/Code-style#parenthesis-should-be-used-liberally-in-expressions-involving-mixed-operators-in-order-to-make-the-precedence-clear

Reply:
Sorry!